lib/rubygems/deprecate.rb

```diff
@@ -41,13 +41,54 @@ def skip_during
     Gem::Deprecate.skip = original
   end
 
+  ##
+  # Simple deprecation method that deprecates +name+ by wrapping it up
+  # in a dummy method. It warns on each call to the dummy method
+  # telling the user of +repl+ (unless +repl+ is :none) and the
+  #Rubygems version that it is planned to go away.
```

Comment:
```suggestion
  # Rubygems version that it is planned to go away.
```

Was that spacing intentional or a typo?

Reply:
Not intentional, thanks @olleolleolle !